specificatie/kadastraal-onroerende-zaken.yaml

```diff
@@ -54,6 +54,26 @@ paths:
             type: "string"
             pattern: "^([a-zA-Z0-9'][a-zA-Z0-9' ,-]*[a-zA-Z0-9]) ([A-IK-Z]{1,2}) ([1-9][0-9]{0,4})( A[1-9][0-9]{0,3})?$"
             example: "'s Gravenhage C 1277 A3"
+        - in: query
+          name: kadastraleAanduidingMetGemeentecode
+          description: |
+                        Kadastrale aanduiding is in sommige domeinen opgeslagen met de gemeentecode in plaats van de gemeentenaam.
+                        Dit is een unieke aanduiding van een onroerende zaak. De volgorde waarin deze string wordt opgebouwd is
+
+                        - gemeentecode van de kadastrale gemeente(http://www.kadaster.nl/schemas/waardelijsten/KadastraleGemeente/).
```

Comment:
Hier ontbreken de vierkante haken als dit als een klikbare link moet worden getoond

Reply:
kadastraleAanduidingMetGemeentecode ook opnemen als mogelijkheid in de description.

Reply:
Haken toegevoegd en description uitgebreid.